src/stack_aggregator.py

```diff
@@ -518,7 +522,7 @@ def get_tr_dependency_data(epv_set):
         result = execute_gremlin_dsl(url=GREMLIN_SERVER_URL_REST, payload=payload)
         if result:
             tr_epv_list['result']['data'] += result['result']['data']
-
```

Comment:
Also it would be good to add logs before and after `execute_gremlin_dsl`, it will help us to know elapsed time for each batch query while debugging

Reply:
Okay